src/DependencyInjection/CmfMenuExtension.php

```diff
@@ -40,7 +40,7 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         if (true === $config['publish_workflow']['enabled']
-            || 'auto' === $config['publish_workflow']['enabled'] && isset($bundles['CmfCoreBundle'])
+            || ('auto' === $config['publish_workflow']['enabled'] && isset($bundles['CmfCoreBundle']))
```

Comment:
no, operator precedence means that && comes before ||. but to avoid such confusion, phpstorm flagged this as risky and proposed to put parenthesis around the expression.